.github/workflows/release.yaml

```diff
@@ -200,6 +200,48 @@ jobs:
       env:
         GH_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
 
+  build_push_docker_image:
+    needs: [build, release]
+    name: Build and push Docker image
+    permissions:
+      contents: read
+      packages: write # To push the Docker image to ghcr.io
+    uses: ./.github/workflows/_build_docker.yaml
+    with:
+      release-tag: ${{ needs.release.outputs.release-tag }}
+
+  publish_docker_imabe_sbom:
+    name: Release the Docker image sbom
+    needs: [build_push_docker_image, release]
+    permissions:
+      contents: write # To publish the SBOM of the Docker image.
+    runs-on: ubuntu-latest
+    env:
+      SYFT_BIN: ${{ github.workspace }}/bin
+      SBOM_FILE_NAME: macaron-sbom-cyclonedx.json
```

Comment:
Please follow the naming convention we use for other SBOMs:
```
macaron-<release-version>-sbom-docker.json
```

e.g., 
`macaron-0.1.0-sbom-docker.json`

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/76/commits/24da909332996f30775f68257a844b0d2b9cbb6a